Mathlib/Tactic/Abel.lean

```diff
@@ -497,37 +498,14 @@ partial def abelNFCore
   withConfig ({ · with zetaDelta := cfg.zetaDelta }) <| go true e
 
 open Parser.Tactic
-/-- Use `abel_nf` to rewrite the main goal. -/
-def abelNFTarget (s : IO.Ref AtomM.State) (cfg : AbelNF.Config) : TacticM Unit := withMainContext do
-  let goal ← getMainGoal
-  let tgt ← withReducible goal.getType'
-  let r ← abelNFCore s cfg tgt
-  if r.expr.isConstOf ``True then
```

Comment:
Flagging for a final look: the new code does `r.expr.consumeMData.isConstOf`.

Reply:
This logic is already being tested: for example this appears in the test file
```lean
example [AddCommGroup α] (a b s : α) : -b + (s - a) = s - b - a := by abel_nf
```
where the fact that `abel_nf` closes the goal is due to this logic.